app/controllers/miq_request_controller.rb

```diff
@@ -400,6 +400,18 @@ def states_for_checkboxes_i18n
     end
   end
 
+  def set_user_options(params)
+    @setDefaults = prov_set_default_options
+    @userDefaults = {
```

Comment:
Prefer snake case in Ruby:
```suggestion
    @set_defaults = prov_set_default_options
    @user_defaults = {
```

Reply:
Also, are these used outside of this method?  If not, then don't use instance vars, and instance use local vars.